terragrunt/aws/ecr/ecr.tf

```
@@ -0,0 +1,41 @@
+resource "aws_ecr_repository" "website_staging_container" {
+  name                 = "website-staging-container"
+  image_tag_mutability = "MUTABLE"
```

One minor tweak here - this can be `IMMUTABLE` since we're not going to need to overwrite any image tags like `latest`:
```suggestion
  image_tag_mutability = "IMMUTABLE"
```

---

Right, also, in the event of another PR being created, it won't overwrite the image. So if 2 PR's are open with content changes you can have 2 different PRs?

---

Yup, absolutely - each PR's changes will be kept separate in their own Docker image and PR review environment.